change/@microsoft-teams-js-690f07e9-cae4-40ab-89e5-b5700ba6ab91.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Initial commit to reset runtimeConfig to minimum for isSupported work",
```

Comment:
Can you add another comment for calling out the "Adding isSupported check to logs capability"?

Reply:
Reminder, you can run 'yarn beachball bump' to see how your changes will appear in the changelog. (I've found it very useful!)

Reply:
Sure.